util/locking.py

```diff
@@ -74,7 +74,7 @@ def acquire(self):
             logger.debug("Acquired lock %s", self._lock_name)
             return True
         except RedisError as re:
-            logger.debug("Could not connect to Redis for lock %s: %s", self._lock_name, re)
+            logger.warn("Could not connect to Redis for lock %s: %s", self._lock_name, re)
```

Comment:
warn has been deprecated, [use warning instead.](https://github.com/python/cpython/blob/3.3/Lib/logging/__init__.py#L1252-L1253) They end up calling the same function but better to be up to date.